### PR TITLE
Allow users to sign up for a new account

### PR DESCRIPTION
**Pull request by aecallahan**
This PR creates a new login module to handle all user login/logout functionality. Register a new account at the `/account/register` endpoint. Login with  your account at `/account/login`. You can also validate your new user was persisted in the database by connecting to your dockerized django app as shown below.
image
User accounts can also be managed more visually by running `python manage.py createsuperuser` to create a valid login at the `/admin` endpoint.
image

Next up: Form input validation and error handling, handling passwords as sha256 hash's instead of plaintext, testing

### CODE REVIEW

File: django/heart_app/settings.py [hunk 1]
```diff
@@ -103,13 +104,16 @@
...
+LOGIN_REDIRECT_URL = '/'
+
 
 # Internationalization
 # https://docs.djangoproject.com/en/3.1/topics/i18n/
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'America/Los_Angeles'
```
Comment (codpro880) [inline django/heart_app/settings.py:116]: How overly brazen of you. I think leaving this as UTC is good?

**Comment by codpro880** [comment]
Have you seen this: https://django-registration.readthedocs.io/en/3.1.1/ ? It ropes in a lot of the boilerplate registration steps. Not saying we have to use it, just wanted to make you aware in case you want to use it. The quick start guide seemed easy enough, anyway

**Review by codpro880 — APPROVED** [review]

**Comment by codpro880** [comment]
Regardless of `django-registration`, merge when ready

File: django/heart_app/settings.py (hunk 1 above)
Comment (aecallahan) [reply django/heart_app/settings.py]: OK switched back

**Comment by aecallahan** [comment]
> Have you seen this: https://django-registration.readthedocs.io/en/3.1.1/ ? It ropes in a lot of the boilerplate registration steps. Not saying we have to use it, just wanted to make you aware in case you want to use it. The quick start guide seemed easy enough, anyway

Seems like a good idea, added

**Pull request merged**